src/components/hero/campaignHero.tsx

```diff
@@ -18,6 +25,12 @@ const CampaignHero = ({ imageID }) => {
     fetchData();
   }, [imageID]);
 
+  const handleError = (e: any) => {
+    console.log(e);
+    setData(defaultFeaturedItemResponse);
+    console.log("data is:", data);
```

Comment:
Nitpick: in the DS we add a prefix to know where the error is coming from. You can still see this in the stack trace but it's helpful if you just read the console message. So something like, "CampaignHero: the data is ...".